advanced_functionality/unity_ml_agents/unity_mlagents_learn.ipynb

```diff
@@ -0,0 +1,211 @@
+{
```

Comment:
<p>A unity build generate many associated files, binary, data folder, .so share library. Would be helpful to be more explicit on what files are required. If I remember correctly, not only the executable file and the data folder, but also the share library </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1382/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
It may depends on the version but the .so file is not exported on my test and the training worked without that. So I think we need only the executable file and the data folder.